transaction/std/src/lib.rs

```diff
@@ -1,12 +1,10 @@
 // Copyright (c) 2018-2020 MobileCoin Inc.
 
-mod block_builder;
 mod error;
 pub mod identity;
 mod input_credentials;
 mod transaction_builder;
 
-pub use block_builder::BlockBuilder;
```

Comment:
i'm glad to see this guy go away, it seems like there wasn't really any uptake here